src/maingoblint.ml

```diff
@@ -36,6 +36,7 @@ let print_help ch =
   fprintf ch " are used instead of double-quotes (\").\n\n";
   fprintf ch "A <jpath> is a path in a json structure. E.g. 'field.another_field[42]';\n";
   fprintf ch "in addition to the normal syntax you can use 'field[+]' append to an array.\n\n";
+  fprintf ch "Some common configurations to start from can be found in conf/examples/*\n";
```

Comment:
This reminds me of something, which becomes more relevant if we want to provide standard confs: currently they only exist in a locally checked out git repository and are not available when `goblint` is directly installed via opam (or installed into the opam switch from git, but then executed in any other directory than the git repo root).
Maybe we should make them part of the installed package and do the CPAchecker-like thing of also looking `--conf` arguments up from that installation directory.